### PR TITLE
Turn urls into links in community.md

### PR DESCRIPTION
**Pull request by jzhuge**

### CODE REVIEW

File: site/docs/community.md [hunk 1]
```diff
@@ -36,13 +36,14 @@ Issues are tracked in GitHub:
...
 
-Iceberg uses Apache's GitHub integration. The code is available at https://github.com/apache/incubator-iceberg
+Iceberg uses Apache's GitHub integration. The code is available at [https://github.com/apache/incubator-iceberg].
 
 The Iceberg community prefers to receive contributions as [Github pull requests][github-pr-docs].
 
 * [View open pull requests][iceberg-prs]
 * [Learn about pull requests][github-pr-docs]
 
+[https://github.com/apache/incubator-iceberg]: https://github.com/apache/incubator-iceberg
```
Comment (rdsr) [inline site/docs/community.md]: I wonder why these links are not showing in the markdown preview. Do you know why?

Comment (rdblue) [reply site/docs/community.md]: Are these proper markdown? I've always used `[text](url)` or `[text][url-ref]`, not `[url]`.

Comment (jzhuge) [reply site/docs/community.md]: @rdsr Somehow mkdocs does not turn this http url is into a link automatically.

Comment (jzhuge) [reply site/docs/community.md]: @rblue This a reference link. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links.

Comment (rdblue) [reply site/docs/community.md]: Using the link text itself for a reference seems odd to me. I think we should add a markdown/mkdocs plugin to linkify URLs instead. There should be a plugin for this.

Comment (jzhuge) [reply site/docs/community.md]: It turned out we can just use Markdown's automatic links. Updated the PR.

**Pull request merged**